docs/docs/50-user-guide/60-reference-docs/80-webhook-receivers/github/index.md

```diff
@@ -0,0 +1,190 @@
+---
+sidebar_label: GitHub
+---
+
+# The GitHub Webhook Receiver
+
+The following instructions will work for Github,
+Github Enterprise Cloud, and GitHub Enterprise Server.
+
+## Configuring the Receiver
+
+The Kargo GitHub webhook receiver will require a Kubernetes Secret. This Secret is required to contain a `secret` key in its data map. You will be required to provide the value assigned to the `secret` key to GitHub in a later step so keep it handy.
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: gh-wh-secret
+  namespace: kargo-demo
+stringData:
+  secret: your-secret-here
+---
+apiVersion: kargo.akuity.io/v1alpha1
+kind: ProjectConfig
+metadata:
+  name: kargo-demo
+  namespace: kargo-demo
+spec:
+  webhookReceivers: 
+    - name: gh-wh-receiver
+      github:
+        secretRef:
+          name: gh-wh-secret
+```
+
+Github will sign payloads using this `secret` via HMAC signature. Our webhook receiver will use this secret to verify the signature.
+
+:::note
+The following command can be used to generate a sufficiently secure secret:
+
+```shell
+echo "$(openssl rand -base64 48 | tr -d '=+/' | head -c 32)"
+```
+
+:::
```

Comment:
It feels awkward to explain what this is and how to generate _after_ the YAML into which this would be inserted.

If we were to lead off with the explanation of how the secret is used and the suggested command for generating it, then the reader has it already in hand by the time they reach the YAML.